### ISSUE TITLE
Space between twisty and label

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode-pull-request-github#2310]
* safari
* open repo in codespaces
* open GH view & expand PRs
* 🧀 some items are indented with whitespace, are icons missing?

<img width="506" alt="Screenshot 2020-12-02 at 14 07 08" src="https://user-images.githubusercontent.com/1794099/100903058-c13af400-34c5-11eb-9f26-8e12ff52ced6.png">

**Comment by RMacfarlane** [comment]
Yeah, this is supposed to be showing user icons, but the content security policy is blocking them.

@matthewisabel Is this something that could be changed on the codespaces side? The icons are urls returned from github, like https://avatars2.githubusercontent.com/u/29464607?v=4&s=64. Or should we try to not render these in the tree when in a codespace?

**Comment by RMacfarlane** [comment]
@matthewisabel @jkeech any thoughts on this?

**Comment by jkeech** [comment]
Adding @legomushroom and @plisy for the CSP rules.

**Comment by legomushroom** [comment]
We own the CSP rules that will block such content, we can allow the icons URL but we want to be very specific about the source. Whos account is this the `29464607` we can allow the image assets from the githubuser content domain for this specific user.

**Comment by RMacfarlane** [comment]
That example happened to be @meganrogge on our team. The idea is that you will test the CSP for those specific image assets and then loosen it to all github users?

**Comment by legomushroom** [comment]
Some issues from `gravatar` hosted images (and potentially other domains) in https://github.com/microsoft/vscode-pull-request-github/issues/2310 cc @eamodio

Sounds like we might want to allow any domains for the image asset, but we need to think about the consequences of doing this. You can say images have a lower attack scope, but still, we saw some instances where malicious actors misuse the assets (at least user tracking is easily possible, also for instance the attacks using the CSS-worklets thru SVG images were possible in the past). Since we want to allow any extension to use any image source, probably we don't have much of a choice here though.

**Comment by RMacfarlane** [comment]
@legomushroom Any updates on this?

**Comment by legomushroom** [comment]
@RMacfarlane no progress on this yet, I've added it to our board so it will be fixed sooner.

**Comment by eamodio** [comment]
This is the same as another issue I filed, for basically the same thing in GitLens.

**Comment by legomushroom** [comment]
@eamodio yeah all extensions will be affected by this.

**Comment by eamodio** [comment]
@legomushroom yeah. I just wanted to point out that other domains also matter, but I see your comment https://github.com/microsoft/vscode-pull-request-github/issues/2310#issuecomment-765762081 above notes that (though the link is wrong).

**Comment by legomushroom** [comment]
Deployed 🚀 Please let me know if you still see the issue 😊
